cognite/extractorutils/extraction_pipelines.py

```diff
@@ -0,0 +1,108 @@
+import logging
```

Comment:
This file is missing copyright and license notice

Reply:
Thanks, will add.